### PR TITLE
4 add support for kafka

### LINKED ISSUE #4: Add support for Kafka
- Provide scripts to install Kafka and performs its integration with Fink
- Add support for Kafka in CI

### PR DESCRIPTION
**Pull request by fjammes**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -50,28 +50,50 @@ jobs:
...
-        run: ./bin/prereq-install.sh
-      - name: Start minikube
+          echo "IMAGE=$(. ./conf.sh && echo $IMAGE)" >> $GITHUB_ENV
+          echo "IMAGE=${IMAGE}"
+          echo "FINK_ALERT_SIMULATOR_DIR=$(. ./conf.sh && echo $FINK_ALERT_SIMULATOR_DIR)" >> $GITHUB_ENV
+      - name: Clone fink-alert-simulator code
+        run: ./bin/clone-fink-alert-simulator.sh
+      - name: Create k8s/kind cluster
+        run: |
+          git clone --depth 1 -b "k8s-1.25.0" --single-branch https://github.com/k8s-school/kind-helper.git
```
Comment (JulienPeloton) [inline .github/workflows/ci.yml:60]: Is `k8s-1.25.0` a version number that will change frequently? Should we put it in a env variable instead?

File: bin/clone-fink-alert-simulator.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+REPO_URL="https://github.com/astrolabsoftware/fink-alert-simulator"
+
+GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+BRANCH=${GHA_BRANCH_NAME:-$GIT_BRANCH}
+# Retrieve same fink-k8s branch if it exists, else use main branch
+if git ls-remote --exit-code --heads "$REPO_URL" "$BRANCH"
+then
+    FINK_ALERT_SIMULATOR_VERSION="$BRANCH"
+else
+    FINK_ALERT_SIMULATOR_VERSION="main"
```
Comment (JulienPeloton) [inline bin/clone-fink-alert-simulator.sh]: The default branch should be `master` instead?

**Review by JulienPeloton — CHANGES_REQUESTED** [review]
Thanks @fjammes for the work! I left some minor comments, and could you also update the README if there are things that need to be documented?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (fjammes) [reply .github/workflows/ci.yml:60]: This does not change frequently and is only used once, so I propose we let it there for now.

File: bin/clone-fink-alert-simulator.sh (hunk 2 above)
Comment (fjammes) [reply bin/clone-fink-alert-simulator.sh]: :+1:

**Comment by fjammes** [comment]
Thanks for you review @JulienPeloton , I plan to add a high level tool to help running the whole stack. So I propose not to document the kafka integration for now in the README, because it should become simpler in the short term.

**Pull request merged**